src/overlays/actors/ovl_En_Nutsball/z_en_nutsball.h

```diff
@@ -4,6 +4,15 @@
 #include "ultra64.h"
 #include "global.h"
 
+#define EN_NUTSBALL_TYPE(thisx) ((thisx)->actor.params)
```

Comment:
```suggestion
#define EN_NUTSBALL_TYPE(thisx) ((thisx)->params)
```
Should take the actor instance right or is that just an MM thing?

Reply:
nothing about params macros are standardized (they are extremely rare in oot so far)
this pr aims to do that, but its been a while since we talked about it https://github.com/zeldaret/oot/pull/1359

Either way, ill change it to this for now

Reply:
It seems that this macro isnt used at all. philtyl used the dns macro in shopnuts.

Im not sure exactly how this should be handled. The attacking actor (shopnuts) is spawned first, and will spawn en_dns after its hit. 
Should the params macros be shared? probably, cause shopnuts just passes its params onto dns. 
Who should be the "owner" of the macros? dns cause he handles the shop? or shopnuts cause its spawned first?

Reply:
Personally I would say they should each have a different copy, since it isn't necessary that they actually share the same params (they just happen to).

Reply:
alright I fixed these up